orochi/client.py

```diff
@@ -685,6 +704,15 @@ def do_status(self, s=''):
             parts.append('(%s)' % track_year)
         print(' '.join(parts) + '.')
 
+        # Log the current song to a file (can be used by other programs)
+        if self._log_current_song is True:
+            cachedir = get_orochi_xdg_dir('XDG_CACHE_HOME', '.cache')
+            filename = os.path.join(cachedir, 'current_song.txt')
+            track_attributes = (track_name, track_performer, track_album, track_year)
+            track_attributes = ['' if v is None else unicode(v) for v in track_attributes]
```

Comment:
Yep, at least on Python 3:

```
(Pdb) [type(x) for x in track_attributes]
[<class 'str'>, <class 'str'>, <class 'str'>, <class 'NoneType'>]
```

I'll simply remove the unicode() call in the merge commit :)